wagtailio/project_styleguide/templates/patterns/base_page.html

```diff
@@ -55,25 +55,13 @@
     <link title="The Wagtail CMS Blog" type="application/atom+xml" rel="alternate" href="{% url 'blog_feed' %}">
     <link title="This Week in Wagtail" type="application/atom+xml" rel="alternate" href="{% url 'newsletter_feed' %}">
 
-    <link rel="apple-touch-icon" sizes="57x57" href="{% static 'img/favicons/apple-touch-icon-57x57.png' %}"/>
-    <link rel="apple-touch-icon" sizes="60x60" href="{% static 'img/favicons/apple-touch-icon-60x60.png' %}"/>
-    <link rel="apple-touch-icon" sizes="72x72" href="{% static 'img/favicons/apple-touch-icon-72x72.png' %}"/>
-    <link rel="apple-touch-icon" sizes="76x76" href="{% static 'img/favicons/apple-touch-icon-76x76.png' %}"/>
-    <link rel="apple-touch-icon" sizes="114x114" href="{% static 'img/favicons/apple-touch-icon-114x114.png' %}"/>
-    <link rel="apple-touch-icon" sizes="120x120" href="{% static 'img/favicons/apple-touch-icon-120x120.png' %}"/>
-    <link rel="apple-touch-icon" sizes="144x144" href="{% static 'img/favicons/apple-touch-icon-144x144.png' %}"/>
-    <link rel="apple-touch-icon" sizes="152x152" href="{% static 'img/favicons/apple-touch-icon-152x152.png' %}"/>
-    <link rel="apple-touch-icon" sizes="180x180" href="{% static 'img/favicons/apple-touch-icon-180x180.png' %}"/>
-    <link rel="icon" type="image/png" href="{% static 'img/favicons/favicon-32x32.png' %}" sizes="32x32"/>
-    <link rel="icon" type="image/png" href="{% static 'img/favicons/android-chrome-192x192.png' %}" sizes="192x192"/>
-    <link rel="icon" type="image/png" href="{% static 'img/favicons/favicon-96x96.png' %}" sizes="96x96"/>
-    <link rel="icon" type="image/png" href="{% static 'img/favicons/favicon-16x16.png' %}" sizes="16x16"/>
-    <link rel="manifest" href="{% static 'img/favicons/manifest.json' %}" />
-    <link rel="shortcut icon" href="{% static 'img/favicons/favicon.ico' %}" />
-    <meta name="msapplication-TileColor" content="#43b1b0"/>
-    <meta name="msapplication-TileImage" content="{% static 'img/favicons/mstile-144x144.png' %}"/>
-    <meta name="msapplication-config" content="{% static 'img/favicons/browserconfig.xml' %}" />
-    <meta name="theme-color" content="#43b1b0"/>
+    {# Reference: https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs #}
+    <link rel="icon" href="{% static 'img/favicons/favicon.ico' %}" sizes="any">{# 32x32 #}
```

Comment:
```suggestion
    <link rel="icon" href="{% static 'img/favicons/favicon.ico' %}" sizes="32x32">{# 32x32 #}
```

as per updated https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

> We need sizes="32x32" for <link> to .ico file in order to fix the [Chrome bug](https://twitter.com/subzey/status/1417099064949235712) where it chooses an ICO file over an SVG.